docs/specs/markdown.yml

```diff
@@ -536,4 +536,21 @@ inputs:
     [some text](#)
 outputs:
   docs/a.json:
+---
+# invalid file link should be outputted as relative path
+inputs:
+  docfx.yml:
+  docs/1/a.md: |
```

Comment:
what if `index.md`/ `../index.md` not found

Reply:
it has been handled, test case added for verification